packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -105,13 +102,13 @@ API calls to the Org creator endpoint (POST `/api/v1/orgs`) have a limit of 10 o
 
 ## Workforce license rate limit multiplier
 
-Workforce orgs that are created after January 7, 2021 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or Single-Sign On).
+Workforce orgs that are created after January 7, 2021 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or single-sign-on).
```

Comment:
```suggestion
Workforce orgs that are created after January 7, 2021 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or Single Sign-On).
```

Reply:
FYI ... I think this was an Acrolinx fix; hopefully won't fail me :o)